packages/wallets/src/features/cfd/screens/MT5TradeScreen/MT5TradeScreen.tsx

```diff
@@ -147,7 +206,16 @@ const MT5TradeScreen: FC<MT5TradeScreenProps> = ({ mt5Account }) => {
                             </Text>
                         </div>
                         <div className='wallets-mt5-trade-screen__description-balance'>
-                            {shouldShowAccountBalance && <Text weight='bold'>{details?.display_balance}</Text>}
+                            {platform !== ctraderPlatform && shouldShowAccountBalance ? (
+                                <Text weight='bold'>{details?.display_balance}</Text>
```

Comment:
@hasan-deriv 
can we use WalletMoney to show balance

Reply:
It doesn't look good 
<img width="379" alt="Screenshot 2024-10-16 at 12 03 36 PM" src="https://github.com/user-attachments/assets/8d023e6f-5fae-43e9-b313-e6a202264ef2">